### PR TITLE
v0.6.0 rc4 fixes 

### PR DESCRIPTION
**Pull request by medgyes**

### CODE REVIEW

File: mitzu/webapp/auth/authorizer.py [hunk 1]
```diff
@@ -347,7 +344,9 @@ def login_local_user(self, email: str, password: str) -> bool:
             return False
 
         token = self._generate_new_token_for_identity(user.id, role=user.role)
-        dash.callback_context.response.set_cookie(self._config.token_cookie_name, token)
+        self.set_cookie(
+            dash.callback_context.response, self._config.token_cookie_name, token
```
Comment (IstvanM) [inline mitzu/webapp/auth/authorizer.py:348]: Why do we use here the DASH callback_context?

File: mitzu/webapp/auth/authorizer.py [hunk 2]
```diff
@@ -358,3 +357,17 @@ def get_current_user_id(self) -> Optional[str]:
         if token_claims is None:
             return None
         return token_claims.get("sub")
+
+    def set_cookie(
+        self,
+        response: werkzeug.wrappers.response.Response,
```
Comment (IstvanM) [inline mitzu/webapp/auth/authorizer.py:363]: Can't we use pure FLASK?
That is a bit more future proof...

File: mitzu/webapp/storage.py [hunk 3]
```diff
@@ -78,7 +78,7 @@ def __init__(
         self,
         connection_string: str = "sqlite://?check_same_thread=False",
     ) -> None:
-        self._engine = SA.create_engine(connection_string)
+        self._engine = SA.create_engine(connection_string, pool_pre_ping=True)
```
Comment (IstvanM) [inline mitzu/webapp/storage.py:81]: Can you explain please what does this do?

File: mitzu/webapp/auth/authorizer.py (hunk 1 above)
Comment (medgyes) [reply mitzu/webapp/auth/authorizer.py:348]: In Flask it's easy to get the current request by calling the `flask.request` but you can't get the current response in a similar way. In dash this is the recommended way to change cookies in the current response https://dash.plotly.com/advanced-callbacks#properties-for-callback_context

File: mitzu/webapp/auth/authorizer.py (hunk 2 above)
Comment (medgyes) [reply mitzu/webapp/auth/authorizer.py:363]: we could, be it seems that there is a bug in mypy: https://github.com/pallets/flask/issues/4976
flask.Response is a subclass of the wekzeug.wrappers.response.Response

File: mitzu/webapp/storage.py (hunk 3 above)
Comment (medgyes) [reply mitzu/webapp/storage.py:81]: with this before executing any queries the sqlalchemy will check the connection and start a new one or recover the current one if there is an error, https://docs.sqlalchemy.org/en/20/core/pooling.html#dealing-with-disconnects

File: mitzu/webapp/auth/authorizer.py (hunk 1 above)
Comment (IstvanM) [reply mitzu/webapp/auth/authorizer.py:348]: Ok for now, but lets think how will we get rid of dash. Will this be possible to remove easily?

**Review by IstvanM — APPROVED** [review]

**Pull request merged**